turbo/cli/flags_zkevm.go

```diff
@@ -177,6 +177,9 @@ func ApplyFlagsForZkConfig(ctx *cli.Context, cfg *ethconfig.Config) {
 		}
 	}
 
+	// set type 1 enabled as a global var for easier access
+	ethconfig.Type1Enabled = ctx.Bool(utils.EnableType1Rollups.Name)
```

Comment:
I doubt about keeping it global rather than a part of the config. This introduces inconsistency with other flags and makes it harder to maintain as a result. Now we ll need to keep track of the zk config and this flag separately, if moved, inherited, or initialized.